modules/prestamosCarpetas/controller/prestamosController.ts

```diff
@@ -418,11 +418,13 @@ async function savePrestamoCarpeta(req, nuevoPrestamo: IPrestamo) {
 }
 
 async function savePrestamosCarpetas(req, nuevosPrestamos: IPrestamo[]) {
+    let promises = [];
     nuevosPrestamos.forEach(async _prestamo => {
```

Comment:
Sugiero usar el `map` en vez de `forEach` como esta en la Guia de derrollo:

https://github.com/andes/andes/wiki/5---Gu%C3%ADa-de-desarrollo#iteraciones-for

Reply:
Hecho!